runtime/src/main/java/org/corfudb/runtime/MultiCheckpointWriter.java

```diff
@@ -94,11 +93,12 @@ public Token appendCheckpoints(CorfuRuntime rt, String author) {
                     author, minSnapshot);
             rt.getObjectsView().TXEnd();
```

Comment:
Is this still needed? In the newly added tests, we're never in a transaction when this is called.

Reply:
Doesn't look like it's needed.